distributor-api-v1/net-pricing.md

```diff
@@ -0,0 +1,128 @@
+# Pricing
+
+**Net price** is an amount without taxes.  
+**Gross price** is a net price + taxes.
+
+In order to use net pricing in distributor, the API returns the [Amount](net-pricing.md#Amount) object as part of responses.
```

Comment:
```suggestion
In order to use pricing in the distributor, the API returns the [Amount](#amount) object as part of responses.
```

the manual is not just for net pricing, but for both net and gross pricing (as per your title).
this is not how links in gitbook work imo

Reply:
do not use newlines inside paragraph, it is either new paragraph or next sentence in the paragraph. Using newlines to style a document is wrong.